lib/viewmodels/home_view_model.dart

```diff
@@ -39,15 +41,21 @@ class HomeViewModel extends AsyncNotifier<List<PostOverview>> {
         // the user repository would have already thrown an exception.
         orElse: () => throw Exception("Owner not found"),
       );
+      final distance = (GeoFirePoint(currentPosition)
+                  .distanceBetweenInKm(geopoint: post.location.geoPoint) *
+              1000)
+          .round(); //TODO: create method because used here and in challenges (+tests)
 
```

Comment:
Indeed I hesitated to switch to `toInt()` to match 
Though it doesn't make much of a difference because we approximate to the meter, I thought using round() would reduce "flicking" as it rounds the double to the closest integer instead of truncating it.
I think the TODO is explicit enough so that the dev in charge of the refactor find all occurrences and then decide of the implementation.
Maybe we can specify this in the future task description to make sure?

Reply:
perfect 👍🏼  we can add it to the task description

Reply:
Created issue #204 for it